dcli/runner.py

```diff
@@ -252,3 +254,7 @@ def solve(path=os.getcwd(), language='empty'):
         error('Challenge type not supported for solve method.')
     clean()
 
+def checkYamlFile(path):
```

Comment:
Your method is doing more than only 'check the yaml file', find a better function name, yesterday I refactored the code to be compliant with python function naming rule. Method must be that way: check_smth. Don't forget to rebase.